numpyro/distributions/mixtures.py

```diff
@@ -288,13 +291,32 @@ class MixtureGeneral(_MixtureBase):
        >>> mixture = dist.MixtureGeneral(mixing_dist, component_dists)
        >>> mixture.sample(jax.random.PRNGKey(42)).shape
        ()
+
+    .. doctest::
+
+       >>> import jax
+       >>> import jax.numpy as jnp
+       >>> import numpyro.distributions as dist
+       >>> mixing_dist = dist.Categorical(probs=jnp.ones(2) / 2.)
+       >>> component_dists = [
+       ...     dist.Normal(loc=0.0, scale=1.0),
+       ...     dist.HalfNormal(scale=0.3),
+       ... ]
+       >>> mixture = dist.MixtureGeneral(mixing_dist, component_dists, support=dist.constraints.real)
+       >>> mixture.sample(jax.random.PRNGKey(42)).shape
+       ()
     """
 
     pytree_data_fields = ("_mixing_distribution", "_component_distributions")
```

Comment:
Could you add `_support` to this tuple?  CI is currently failing.

Reply:
Done